Resources/Locale/en-US/guidebook/chemistry/plant-attributes.ftl

```diff
@@ -1,5 +1,7 @@
 plant-attribute-growth = age
 plant-attribute-water = water level
+plant-attribute-seeds = seeds availability
```

Comment:
`plant-attribute-seeds` is no longer used and can be removed.

Reply:
Oops part 3! Thanks for the extensive review.